coldfront/plugins/qumulo/tasks.py

```diff
@@ -93,25 +93,27 @@ def conditionally_update_storage_allocation_statuses() -> None:
 def ingest_quotas_with_daily_usage() -> None:
     logger = logging.getLogger("task_qumulo_daily_quota_usages")
 
-    quota_usages = __get_quota_usages_from_qumulo(logger)
-    __set_daily_quota_usages(quota_usages, logger)
-    __validate_results(quota_usages, logger)
-
-
-def __get_quota_usages_from_qumulo(logger):
     qumulo_api = QumuloAPI()
-    quota_usages = qumulo_api.get_all_quotas_with_usage()
-    return quota_usages
+    quota_usages = qumulo_api.get_all_quotas_with_usage()["quotas"]
+    base_allocation_quota_usages = list(
+        filter(
+            lambda quota_usage: AclAllocations.is_base_allocation(quota_usage["path"]),
```

Comment:
@harterjwustl 
During the story refinement meeting, it was mentioned that sub-allocations have a parent through the allocation linkage. 

Meanwhile, @peaches7083 mentioned that some sub-allocations were created outside the parent allocation and could not be linked back to the parent through the allocation linkage. If the intention is to use the Allocation Linkage, the code should reflect it. If the allocation linkage is not found, could you try using the sub-string path?

Reply:
I'm going to work on this.  Note the discussion in appeng_only that there are not any back-end validation preventing some misuse or bad data going through to the database, so it might not end up super clean

Reply:
We discussed this in slack:  This is a good idea, but I have gone ahead and moved this into a new ticket to reduce scope creep.

https://washu.atlassian.net/browse/ITDEV-35716